### ISSUE TITLE
Наложение звуков

### ISSUE THREAD

**Issue by ikamensh** [ikamensh/drim_gam#15]
@reef425 
1.Насколько я понимаю сейчас если проигрываются 2 и более звука одновременно, то либо первый либо последний слышен, остальные - нет. Можем ли мы изменить систему звука так чтобы одновременно могло звучать до x звуков? 

2. Есть ли в qt параметр громкости? В идеале было бы хорошо иметь возможность послать вместе со звуком параметр громкости, таким образом громкость звуков регулировалась бы игровой логикой.

**Comment by reef425** [comment]
Да, звук нужно подправить. Займусь как закончу генератор уровней.

чт, 25 окт. 2018 г. в 11:56, Ilya Kamenshchikov <notifications@github.com>:

**Comment by ikamensh** [comment]
@reef425 
Какая ситуация со звуком? можем ли закрыть этот тред?
